build.gradle

```diff
@@ -13,38 +13,44 @@
  * License for the specific language governing permissions and limitations under
  * the License.
  */
+
 task wrapper(type: Wrapper) {
-    gradleVersion = '1.11'
+    gradleVersion = '2.2.1'
 }
 
 buildscript {
-    def rosMavenPath = "$System.env.ROS_MAVEN_PATH".split(':').collect { 'file://' + it }
-    def rosMavenRepository = "$System.env.ROS_MAVEN_REPOSITORY"
-    repositories {
-        rosMavenPath.each { p ->
-            maven {
-                url p
-            }
-        }
-        mavenLocal()
-        maven {
-            url rosMavenRepository
-        }
-    }
+    apply from: "https://github.com/talregev/rosjava_mvn_repo/raw/master/buildscript.gradle"
+    //noinspection GroovyAssignabilityCheck
     dependencies {
-        classpath group: 'org.ros.rosjava_bootstrap', name: 'gradle_plugins', version: '[0.1,0.2)'
+        classpath "com.android.tools.build:gradle:1.0.1"
     }
 }
 
 apply plugin: 'catkin'
 
 allprojects {
+    apply from: "https://github.com/talregev/rosjava_mvn_repo/raw/master/maven.gradle"
     group 'com.github.rosjava.android_apps'
     version = project.catkin.pkg.version
 }
 
 subprojects {
     apply plugin: 'ros-android'
+
+    afterEvaluate { project ->
+        android {
+
+            packagingOptions {
+                /* https://github.com/rosjava/android_core/issues/194 */
+                exclude 'META-INF/LICENSE.txt'
+                exclude 'META-INF/NOTICE.txt'
+                exclude 'META-INF/gradle-plugins/catkin.properties'
```

Comment:
I am not really sure,
It start to appear when you updated bootstrap. 
are you try to compile this project with catkin_make?